inspirehep/modules/theme/bundles.py

```diff
@@ -65,7 +66,8 @@
         "clipboard": "~1.5.8",
         "flightjs": "~1.5.0",
         "angular": "~1.4.9",
-        "angular-sanitize": "~1.4.9"
+        "angular-sanitize": "~1.4.9",
+        "angular-ui-bootstrap": "~1.2.4"
```

Comment:
Checking now we don't need to add it. As our library `inspirehep-js` depends on it (has it on its `package.json`) it already gets installed. And it is better since we can control the version from that package.